lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -34,6 +36,8 @@ def commands
       "db_connect"    => "Connect to an existing database",
       "db_disconnect" => "Disconnect from the current database instance",
       "db_status"     => "Show the current database status",
+      "db_save"       => "Save the current database connection so it is re-used on startup",
```

Comment:
Minor: consider using the description from `cmd_db_save_help` it is also about data service and not database.